shared/message/vision/Ball.proto

```diff
@@ -34,20 +34,24 @@ message Ball {
     }
     message Measurement {
         MeasurementType type = 1;
-        // Vector from camera to ball in camera coordinates
+        /// Vector from camera to ball in camera coordinates
```

Comment:
Is this also in spherical coordinates with inverse distance? `(1/r, theta, phi)`?

Reply:
These might have changed with the changes to the ball and goal detectors. Check if they are the `s...` form vectors now with reciprocal spherical values